qiime2/sdk/tests/test_plugin_manager.py

```diff
@@ -60,6 +68,36 @@ def test_importable_types(self):
 
     # TODO: add tests for type/directory/transformer registrations
 
+    def test_importable_formats(self):
+        obs = self.pm.importable_formats
+        exp = {
+            'IntSequenceDirectoryFormat': IntSequenceDirectoryFormat,
+            'MappingDirectoryFormat': MappingDirectoryFormat,
+            'IntSequenceV2DirectoryFormat': IntSequenceV2DirectoryFormat,
+            'IntSequenceFormatV2': IntSequenceFormatV2,
+            'FourIntsDirectoryFormat': FourIntsDirectoryFormat,
+            'IntSequenceFormat': IntSequenceFormat
+        }
+        self.assertEqual(set(obs), set(exp))
+
+    def test_importable_formats_with_dummy(self):
+        obs = self.pm.importable_formats
+        self.assertFalse('UnimportableFormat' in obs.keys())
```

Comment:
Minor: when checking if a key exists you can just use `in` directly, e.g. `'UnimportableFormat' in obs`. Even shorter, you can use `self.assertNotIn`:

```python
self.assertNotIn('UnimportableFormat', obs)
self.assertNotIn('UnimportableDirectoryFormat', obs)
```